src/Codeception/Lib/Actor/Shared/Pause.php

```diff
@@ -94,8 +94,8 @@ public function pause()
         $readline->addMapping($tput->get('key_f10'), $clearStashFn);
 
         $output->writeln("  <comment>Execution PAUSED, starting Interactive Shell...</comment>");
-        $output->writeln("  Type in commands to try them:");
-        $output->writeln("  - <info>ENTER</info> to continue");
+        $output->writeln("  Type in PHP commands to try them, or press:");
```

Comment:
`PHP Commands` is not accurate, what you actually have to enter is the name of actor method and the list of parameters, e.g. `amOnPage('/info')`,  because command is executed by this code:
```php
 $value = eval("return \$I->$command;");
```

Clearly command injection is possible with semicolon, but it doesn't look like intended use case for me.

Reply:
I thought the trailing `;` is mandatory... Now removed "PHP" again :-)